public/about.html

```diff
@@ -41,25 +44,25 @@
             <div class="collapse navbar-collapse" id="navbarCollapse">
                <ul class="navbar-nav mr-auto">
                   <li class="nav-item dropdown">
-                     <a class="nav-link dropdown-toggle" href="#" id="aboutDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">About</a>
+                     <a class="nav-link dropdown-toggle" href="#" id="aboutDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button">About</a>
                      <div class="dropdown-menu" aria-labelledby="aboutDropdownMenuLink">
                         <a class="dropdown-item" href="about.html">Project Overview</a>
                         <a class="dropdown-item" href="people.html">Team</a>
                         <a class="dropdown-item" href="partner.html">Partner Projects</a>
                         <a class="dropdown-item" href="media.html">Media Coverage</a>
                         <div class="dropdown-divider"></div>
-                        <a class="dropdown-item" href="contact.html">Contact Us</a>
+                        <a class="dropdown-item" href="contact.html" role="button">Contact Us</a>
```

Comment:
This one doesn't need a role

Reply:
fixed in e42d2addcf2e53fd2b019d3b75afb3cc9fab9635